definitions/mongodb.rb

```
@@ -23,25 +23,25 @@
     :bind_ip => nil, :port => 27017 , :logpath => "/var/log/mongodb",
     :dbpath => "/data", :configserver => [],
     :replicaset => nil, :enable_rest => false, :smallfiles => false, :notifies => [] do
-    
+
   include_recipe "mongodb::default"
-  
+
   name = params[:name]
   type = params[:mongodb_type]
   service_action = params[:action]
   service_notifies = params[:notifies]
-  
+
   bind_ip = params[:bind_ip]
   port = params[:port]
 
   logpath = params[:logpath]
   logfile = "#{logpath}/#{name}.log"
-  
+
   dbpath = params[:dbpath]
-  
-  configfile = node['mongodb']['configfile']
+
+  configfile = if node['mongodb']['configfile'].nil? then "/etc/mongodb.conf" else node['mongodb']['configfile'] end
```

default values should go to attributes instead of hiding inside of logic in the recipe.